symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/pagination/model/StreamPaginationAttribute.java

```diff
@@ -0,0 +1,18 @@
+package com.symphony.bdk.core.service.pagination.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Accessors(fluent = true)
+public class StreamPaginationAttribute {
```

Comment:
I am not fond of having duplicated classes. IMHO having only PaginationAttribute class should be fine and not confusing.

Reply:
you mean that the `PaginationAttribute` will contains all of skip, limit, chunksize, totalsize?

Reply:
Sorry I only saw two integers... Field names are different indeed.